configure

```diff
@@ -792,7 +792,6 @@ infodir
 docdir
 oldincludedir
 includedir
-runstatedir
```

Comment:
These `runstatedir` changes occur because your autoconf doesn't contain some long-standing unreleased patches in autoconf (Cygwin has the same problem). These deletions should not be committed - I use `git checkout -p configure` on Cygwin to get rid of them quickly